packages/ddp-client/livedata_connection.js

```diff
@@ -38,7 +38,11 @@ var Connection = function (url, options) {
     reloadWithOutstanding: false,
     supportedDDPVersions: DDPCommon.SUPPORTED_DDP_VERSIONS,
     retry: true,
-    respondToPings: true
+    respondToPings: true,
+    // When updates are coming within this ms interval, batch them together.
+    bufferedWritesInterval: 5,
+    // Flush buffers immediately if writes are happening continuously for more than this many ms.
+    bufferedWritesMaxAge: 500
```

Comment:
How would one override these options for the default Meteor connection? I can imagine that 500ms might be too long for some apps that have basically continuous updates. (Maybe those don't exist?)